Adding context to build_after (#7713)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/freshness.md

models:
  [<resource-path>](/reference/resource-configs/resource-path):
    [+](/reference/resource-configs/plus-prefix)[freshness](/reference/resource-properties/freshness):

>       build_after:  # build this model no more often than every X amount of time, as long as as it has new data. Available only for state aware orchestration on the dbt platform. Not applicable to dbt Core. 

Author: ```suggestion
      build_after:  # build this model no more often than every X amount of time, as long as it has new data. Available only on dbt platform Enterprise tiers. 
```

  - name: stg_orders
    config:
      freshness:

>         build_after:  # build this model no more often than every X amount of time, as long as as it has new data. Available only on dbt platform Enterprise tiers. 

Reviewer: ```suggestion
        build_after:  # build this model no more often than every X amount of time, as long as it has new data. Available only on dbt platform Enterprise tiers. 
```

>         build_after:  # build this model no more often than every X amount of time, as long as as it has new data. Available only for state aware orchestration on the dbt platform. 

Author: ```suggestion
        build_after:  # build this model no more often than every X amount of time, as long as as it has new data. Available only on dbt platform Enterprise tiers. 
```

| Configuration | Description |
|--------------|-------------|

> | `build_after` | Available for the dbt platform only. Config nested under `freshness`. Used to determine whether a model should be rebuilt when new data is present, based on whether the specified count and period have passed since the model was last built. Although dbt checks for new data every time the job runs, `build_after` ensures the model is only rebuilt if enough time has passed and new data is available. |

Author: ```suggestion
| `build_after` | Available on dbt platform Enterprise tiers only. Config nested under `freshness`. Used to determine whether a model should be rebuilt when new data is present, based on whether the specified count and period have passed since the model was last built. Although dbt checks for new data every time the job runs, `build_after` ensures the model is only rebuilt if enough time has passed and new data is available. |
```
